canvas_admin_tools/settings/base.py

```diff
@@ -136,6 +139,22 @@
     }
 }
 
+# RQ
+# http://python-rq.org/docs/
+
+RQ_QUEUES = {
+    'default': {
+        'HOST': REDIS_HOST,
+        'PORT': REDIS_PORT,
+        'DB': 0,
```

Comment:
I went digging around for more info on this `'DB': 0` thing and went down a rabbit hole quickly. Can you explain what it signifies and how it connects to django-redis-cache config, etc? Thanks.